Transfer with tokens is not being received by the receiver / not being submitted properly

### Description

Opening this as a separate issue from https://github.com/rucio/rucio/issues/7750 as most of the tests failing under #7750 were fixed by properly migrating to the FTS >3.14.0 ActiveMQ daemon, but `test_transfer_with_tokens` is still failing. 

Trying to debug this, it seems like the transfer is not being properly submitted. In the FTS container's logs, there is no folder created for the WEB1->XRD5 transfers (while e.g. `xrd3__xrd4` is created by `test_receiver_archiving)`, and the `fts3server.log` and `msg.log` (the ActiveMQ log flie) don't report any activity, even though they do for `test_receiver_archiving`.

The overall test setup is the same as `test_receiver_archiving`, while the RSEs used for SRC and DST are the same as the ones used in `test_bittorrent_submission`, and both of these test work fine. I think `test_receiver_archiving` is the best one to take into account as reference for the most similar one that passes. For some reason, it seems that using WEB1 as the RSE is causing this overall transfer to not work.

The full logs are:
```
=================================== FAILURES ===================================
_ test_transfer_with_tokens[file_config_mock{'oidc': {'admin_issuer': 'indigoiam'}}] _
[gw0] linux -- Python 3.9.21 /opt/venv/bin/python
tests/test_conveyor.py:1665: in test_transfer_with_tokens
    assert False  # Waited too long; fail the test
E   assert False
------------------------------ Captured log call -------------------------------
DEBUG    root:uploadclient.py:347 Num. of files that upload client is processing: 1
DEBUG    urllib3.connectionpool:connectionpool.py:1021 Starting new HTTPS connection (1): rucio:443
DEBUG    urllib3.connectionpool:connectionpool.py:474 https://rucio:443 "GET /rses/?expression=WEB1 HTTP/1.1" 200 None
DEBUG    root:uploadclient.py:375 Input validation done.
INFO     root:uploadclient.py:383 Preparing upload for file CQKZHNFLCB
DEBUG    urllib3.connectionpool:connectionpool.py:474 https://rucio:443 "GET /rses/WEB1/attr/ HTTP/1.1" 200 226
DEBUG    root:uploadclient.py:427 wan domain is used for the upload
DEBUG    root:uploadclient.py:689 Registering file
DEBUG    urllib3.connectionpool:connectionpool.py:474 https://rucio:443 "GET /accounts/root/scopes/ HTTP/1.1" 200 51
WARNING  root:uploadclient.py:698 Scope mock not found for the account root.
DEBUG    root:uploadclient.py:723 Skipping dataset registration
DEBUG    urllib3.connectionpool:connectionpool.py:474 https://rucio:443 "GET /dids/mock/file_0ace9e41acac44708b0c6d1e3c30066d/meta?plugin=DID_COLUMN HTTP/1.1" 404 138
INFO     root:logging.py:419 [0/1]: Daemon started
DEBUG    root:logging.py:419 [0/1]: Starting next iteration
INFO     root:logging.py:419 [0/1]: Got 1 transfers for None in 0.014373575000035999 seconds
DEBUG    root:logging.py:419 [0/1]: Only 1 transfers for None which is less than bulk 100
INFO     root:logging.py:419 [0/1]: Heartbeat cleaned up
DEBUG    root:transfer.py:1237 d44fe1acf1ce468dbe44bb5e8e8ee227(mock:file_0ace9e41acac44708b0c6d1e3c30066d): Working on 1 sources: WEB1:0:1
INFO     root:transfer.py:1297 d44fe1acf1ce468dbe44bb5e8e8ee227(mock:file_0ace9e41acac44708b0c6d1e3c30066d): 1 ordered sources: WEB1:0:0:1:0
DEBUG    root:fts3.py:981 OAuth2/OIDC available for transfer ['WEB1--d44fe1acf1ce468dbe44bb5e8e8ee227->XRD5']
DEBUG    root:fts3.py:906 Failed to set up any fts3 archive-metadata plugins: No section: 'transfers'
DEBUG    urllib3.connectionpool:connectionpool.py:1021 Starting new HTTPS connection (1): indigoiam:443
DEBUG    urllib3.connectionpool:connectionpool.py:474 https://indigoiam:443 "POST /token HTTP/1.1" 200 None
INFO     root:fts3.py:915 Using a token to authenticate with FTS instance fts
DEBUG    root:submitter.py:163 Starting to group transfers https://fts:8446
DEBUG    root:fts3.py:369 RSE:XRD5 Is it Tape? False overwrite_when_only_on_disk:False overwrite:True overwrite_corrupted_files=False
DEBUG    root:fts3.py:376 RSE attributes are: {'XRD5': True, 'oidc_support': True, 'bittorrent_driver': 'qbittorrent', 'qbittorrent_management_address': 'https://xrd5:8098/', 'bittorrent_tracker_addr': 'http://xrd5:10001/announce', 'fts': 'https://fts:8446'}
INFO     root:fts3.py:242 No common checksum method for WEB1--d44fe1acf1ce468dbe44bb5e8e8ee227->XRD5. Verifying destination only.
DEBUG    root:fts3.py:463 Job parameters are : {'account': root, 'verify_checksum': 'target', 'copy_pin_lifetime': None, 'bring_online': None, 'job_metadata': {'issuer': 'rucio', 'multi_sources': False, 'overwrite_when_only_on_disk': False}, 'overwrite': True, 'overwrite_when_only_on_disk': False, 'priority': 3, 's3alternate': True, 'max_time_in_queue': 168}
DEBUG    root:fts3.py:502 bulk_group_transfers: Job parameters are: {'account': root, 'verify_checksum': 'target', 'copy_pin_lifetime': None, 'bring_online': None, 'job_metadata': {'issuer': 'rucio', 'multi_sources': False, 'overwrite_when_only_on_disk': False}, 'overwrite': True, 'overwrite_when_only_on_disk': False, 'priority': 3, 's3alternate': True, 'max_time_in_queue': 168}
DEBUG    root:fts3.py:548 bulk_group_transfers: grouped_transfers.values(): Job parameters are: {'account': root, 'verify_checksum': 'target', 'copy_pin_lifetime': None, 'bring_online': None, 'job_metadata': {'issuer': 'rucio', 'multi_sources': False, 'overwrite_when_only_on_disk': False}, 'overwrite': True, 'overwrite_when_only_on_disk': False, 'priority': 3, 's3alternate': True, 'max_time_in_queue': 168}
DEBUG    root:submitter.py:168 Starting to submit transfers for https://fts:8446
DEBUG    root:submitter.py:170 submitjob: transfers=['WEB1--d44fe1acf1ce468dbe44bb5e8e8ee227->XRD5'], job_params={'account': root, 'verify_checksum': 'target', 'copy_pin_lifetime': None, 'bring_online': None, 'job_metadata': {'issuer': 'rucio', 'multi_sources': False, 'overwrite_when_only_on_disk': False}, 'overwrite': True, 'overwrite_when_only_on_disk': False, 'priority': 3, 's3alternate': True, 'max_time_in_queue': 168}
DEBUG    root:transfer.py:343 PREPARING REQUEST d44fe1acf1ce468dbe44bb5e8e8ee227 DID mock:file_0ace9e41acac44708b0c6d1e3c30066d TO SUBMITTING STATE PREVIOUS None FROM ['davs://web1:443/rucio/mock/78/be/file_0ace9e41acac44708b0c6d1e3c30066d'] TO davs://xrd5:1098//rucio/mock/78/be/file_0ace9e41acac44708b0c6d1e3c30066d USING https://fts:8446 
DEBUG    root:common.py:445 About to submit job to https://fts:8446 with timeout None
DEBUG    urllib3.connectionpool:connectionpool.py:1021 Starting new HTTPS connection (1): indigoiam:443
DEBUG    urllib3.connectionpool:connectionpool.py:474 https://indigoiam:443 "POST /token HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:1021 Starting new HTTPS connection (1): indigoiam:443
DEBUG    urllib3.connectionpool:connectionpool.py:474 https://indigoiam:443 "POST /token HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:1021 Starting new HTTPS connection (1): fts:8446
DEBUG    urllib3.connectionpool:connectionpool.py:474 https://fts:8446 "POST /jobs HTTP/1.1" 200 50
DEBUG    root:common.py:472 Submit job b36171e0-ab7d-11f0-8859-7a4f9651b36b to https://fts:8446 in 0.2125122370000554 seconds
INFO     root:transfer.py:432 Setting state(SUBMITTED), transfertool(fts3), external_host(https://fts:8446) and eid(b36171e0-ab7d-11f0-8859-7a4f9651b36b) for transfers: d44fe1acf1ce468dbe44bb5e8e8ee227
DEBUG    root:transfer.py:437 COPYING REQUEST d44fe1acf1ce468dbe44bb5e8e8ee227 DID mock:file_0ace9e41acac44708b0c6d1e3c30066d USING https://fts:8446 with state(RequestState.SUBMITTED) with eid(b36171e0-ab7d-11f0-8859-7a4f9651b36b)
DEBUG    root:transfer.py:509 Finished to register transfer state for b36171e0-ab7d-11f0-8859-7a4f9651b36b
DEBUG    root:logging.py:419 [0/1]: Heartbeat renewed
DEBUG    root:logging.py:419 [0/1]: Heartbeat renewed
DEBUG    root:logging.py:419 [0/1]: Heartbeat renewed
INFO     root:logging.py:419 [0/1]: Heartbeat cleaned up
```

from https://github.com/rucio/rucio/actions/runs/18599299319/job/53034376598?pr=7899

### Steps to reproduce

Execute test_transfer_with_tokens

### Rucio Version

_No response_

### Additional Information

_No response_

--- a/etc/docker/dev/fts/entrypoint.sh
+++ b/etc/docker/dev/fts/entrypoint.sh
@@ -39,4 +39,5 @@ update-ca-trust
 /usr/sbin/fts_server               # main FTS server daemonizes
 /usr/sbin/fts_qos                  # for the stager tests
 /usr/sbin/fts_activemq             # daemon to send messages to activemq
+/usr/sbin/fts_token                # daemon to manage token
 /usr/sbin/httpd -DFOREGROUND       # FTS REST frontend & FTSMON

--- a/tests/test_conveyor.py
+++ b/tests/test_conveyor.py
@@ -1624,7 +1624,6 @@ def test_checksum_validation(rse_factory, did_factory, root_account):
 
 
 @skip_rse_tests_with_accounts
-@pytest.mark.skip(reason="Pending https://github.com/rucio/rucio/issues/8133")
 @pytest.mark.needs_iam
 @pytest.mark.noparallel(groups=[NoParallelGroups.XRD, NoParallelGroups.SUBMITTER, NoParallelGroups.RECEIVER])
 @pytest.mark.parametrize("file_config_mock", [

